spec/libtest/test_subject.h

```diff
@@ -51,6 +51,26 @@ typedef struct _TestStruct {
  * TestSubject:
  *
  * Main class used to test all sort of things directly related or not to GObject.
+ * 
+ * Example docs for testing:
+ *
+ * getter: [method@Test.Subject.get_crystallize_docs]
+ * 
+ * setter: [method@Test.Subject.set_crystallize_docs]
+ * 
+ * is: [method@Test.Subject.is_crystallize_docs]
```

Comment:
git show show some extra white spaces here
![image](https://user-images.githubusercontent.com/727809/157916146-ba027f38-dda0-45e5-b1ac-b3e18f094db2.png)

I thought github would show then in the web interface... :confused:

Reply:
You can use `test_subject_is_bool` on this instead of create an empty funcion.